src/libs/Navigation/AppNavigator/ModalStackNavigators.js

```diff
@@ -18,17 +18,24 @@ const defaultSubRouteOptions = {
  */
 function createModalStackNavigator(screens) {
     const ModalStackNavigator = createStackNavigator();
-    return () => (
-        <ModalStackNavigator.Navigator screenOptions={defaultSubRouteOptions}>
-            {_.map(screens, (getComponent, name) => (
-                <ModalStackNavigator.Screen
-                    key={name}
-                    name={name}
-                    getComponent={getComponent}
-                />
-            ))}
-        </ModalStackNavigator.Navigator>
-    );
+
+    function ModalStack() {
```

Comment:
pardon my ignorance, but did we need to update the anon methods too?

Reply:
Indeed, that would be required because otherwise the new eslint rule added would throw error:
<img width="728" alt="Screenshot 2023-10-19 at 21 08 30" src="https://github.com/Expensify/App/assets/56457735/7685c5ed-f841-4627-b7a8-8d1c1ea340e6">